db/schema.rb

```diff
@@ -142,6 +142,7 @@
     t.text "slug"
     t.string "social_image"
     t.integer "spaminess_rating", default: 0
+    t.string "text_lang"
```

Comment:
Sorry that these two lines in this file are unrelated changes which I failed to exclude.